src/commands/handlers/misc.js

```diff
@@ -300,6 +298,12 @@ const handlers = {
         cache.destroy();
     },
 
+    FAIL: standardReply,
```

Comment:
I'm actually okay with adding this, but I get the sentiment for now at least it should follow the other style guide. 

for example : 

```
FAIL: function(command, handler) {
 const [command, code, ...context] = command.params;
   ...
}
```

The naming convention is close but just for consistency we can use `command`

`description` is also obligatory for the server side, so it probably should also be added. 

Also while `context` is in the spec, it should be documented for the IRC3 implementation here too.

For brevity the spec is listed here : https://ircv3.net/specs/extensions/standard-replies

It would be nice to also add the documentation of it somewhere.